openmdao/docs/basic_guide/first_optimization.rst

```diff
@@ -63,7 +62,7 @@ Defining the Design Variables and Objective
 
 Next, we set up the problem formulation so that the optimizer knows what to vary and which objective to optimize.
 In these calls, we are always going to be specifying a single variable. For :ref:`add_design_var <feature_add_design_var>`,
-the variable will always be the output of an :ref:`IndepVarComp <comp-type-1-indepvarcomp>`.
+the variable will always be a component input. If the input is promoted, then it will be the promoted name.
```

Comment:
Will it always be a component input?  Should we say 'typically' instead since it can still be an IndepVarComp output in some cases?

Reply:
I have removed the word 'always' from here, and also in the previous sentence about objectives.  I don't know how much we want to confuse them about things in their first optimization tutorial.

(I think these are all going to get re-written anyway.)